src/controller/org.controller/org.controller.js

```diff
@@ -26,10 +29,17 @@ async function getOrgs (req, res, next) {
           time: true
         }
       }
-    ])
+    ]
+
+    const pg = await repo.aggregatePaginate(agt, options)
+    const count = await repo.countDocuments({})
+
+    if (count >= CONSTANTS.PAGINATOR_OPTIONS.limit) {
+      utils.setPaginatorHeaders(res, pg)
+    }
 
     logger.info({ message: 'The orgs were sent to the user.' })
-    return res.status(200).json(result)
+    return res.status(200).json(pg.itemsList)
```

Comment:
All lists should be returned as a named array. So for this one it should be `orgs` or `organizations`.